cmd/notation/list.go

```diff
@@ -112,5 +112,9 @@ func printSignatureManifestDigests(ctx context.Context, targetDesc ocispec.Descr
 		// print last signature digest
 		fmt.Printf("    └── %s\n", prevDigest)
 	}
+
+	if !titlePrinted {
+		fmt.Printf("%s has no associated signature\n", ref)
+	}
```

Comment:
> Yes, otherwise the output (as shown below) looks incomplete.
> 
> For unsigned image referred by tag, user wont know which image was inspected/listed.
> 
> ```shell
> # unsigned image referred by tag
> ➜  notation git:(no-sig) ✗ ./notation inspect $IMAGE 
> Warning: Always inspect the artifact using digest(@sha256:...) rather than a tag(:v1) because resolved digest may not point to the same signed artifact, as tags are mutable.
> ```
> 
> For unsigned image referred by digest, not printing anything doesn't looks right and would like to keep output consistent with unsigned image refereed by tag
> 
> ```shell
> # unsigned image referred by digest
> ➜  notation git:(no-sig) ✗ ./notation inspect $IMAGE 
> ```

Got it. Makes sense to me.